recipe/meta.yaml

```diff
@@ -42,7 +44,10 @@ about:
     effectiveness of tests. It can show which parts of your code are being
     exercised by tests, and which are not.
   doc_url: http://coverage.readthedocs.org
-  dev_url: https://bitbucket.org/ned/coveragepy
+  # the old dev_url is deprecated dev_url: https://bitbucket.org/ned/coveragepy
```

Comment:
There's no need to document the old `dev_url` in a comment. If anyone needs to know, it can be looked up via `git log`.

Reply:
The unnecessary comments were removed:

https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L42&L53